q2_types/feature_data/_transformer.py

```diff
@@ -359,3 +359,18 @@ def _19(data: AlignedDNAIterator) -> AlignedDNAFASTAFormat:
 @plugin.register_transformer
 def _33(ff: AlignedDNAFASTAFormat) -> qiime2.Metadata:
     return _dnafastaformats_to_metadata(ff)
+
+
+# differential types
+@plugin.register_transformer
+def _222(ff: DifferentialFormat) -> pd.DataFrame:
+    df = pd.read_csv(str(ff), sep='\t', comment='#', skip_blank_lines=True,
+                     header=0, dtype=object, index_col=0)
+    return df
+
+
+@plugin.register_transformer
```

Comment:
If my comment above re Metadata is applicable, this transformer could be re-implemented as shown [here](https://github.com/qiime2/q2-demux/blob/master/q2_demux/_transformer.py#L120-L124).